packages/service/tests/Service.test.ts

```diff
@@ -290,16 +289,30 @@ describe(
             expect(tx.isRgbpp).toBeTypeOf('boolean');
             if (tx.isRgbpp) {
               expect(tx.isomorphicTx).toBeDefined();
-              expect(tx.isomorphicTx.status.confirmed).toBeTypeOf('boolean');
-              const hasTxOrVirtualTx = tx.isomorphicTx.ckbVirtualTx ?? tx.isomorphicTx.ckbTx;
+              expect(tx.isomorphicTx?.status.confirmed).toBeTypeOf('boolean');
+              const hasTxOrVirtualTx = tx.isomorphicTx?.ckbVirtualTx ?? tx.isomorphicTx?.ckbTx;
               if (hasTxOrVirtualTx) {
-                expect(tx.isomorphicTx.inputs).toBeDefined();
-                expect(tx.isomorphicTx.outputs).toBeDefined();
+                expect(tx.isomorphicTx?.inputs).toBeDefined();
+                expect(tx.isomorphicTx?.outputs).toBeDefined();
               }
             }
           }
         }
       });
+      it('getRgbppAssetInfoByTypeScript()', async () => {
```

Comment:
Maybe add a test case for new compatible UDT?

Reply:
The `btc-assets-api` will support compatible UDTs after upgrading `rgbpp-sdk` to the latest version.